lib/gold_miner.rb

```diff
@@ -16,10 +16,10 @@ def mine_in(slack_channel, slack_client: GoldMiner::Slack::Client, env_file: ".e
       .fmap { |client| explore(slack_channel, client) }
   end
 
-  def convert_messages_to_blogpost(channel, messages, blog_post_builder: GoldMiner::BlogPost)
+  def convert_messages_to_blogpost(channel, gold_nuggets, blog_post_builder: GoldMiner::BlogPost)
```

Comment:
I know the method name still contains `messages`, but I'll update this as part of a different PR.

Reply:
Also, `channel` is not needed here anymore. `gold_nuggets` contain everything `BlogPost` needs

Reply:
We still need `channel` to create the blog post name ("This week in #{channel}"). It's a weird coupling, but I'll solve that in the future.